Source/Core/Core/ConfigManager.h

```diff
@@ -95,7 +95,7 @@ struct SConfig : NonCopyable
   bool bDSPThread = false;
   bool bDSPHLE = true;
   bool bSyncGPUOnSkipIdleHack = true;
-  bool bForceNTSCJ = false;
+  bool bSelectNTSCAsJapanese = false;
```

Comment:
I don't think it's a good name as well. I would rather say JapaneseNTSC even if that's not entirely true. AFAIK, Japan is not the only country using NTSC-J display and is probably not the only country having NTSC-J games I guess. ```bForceNTSCJ``` seems fine to me.